### PR TITLE
Update video2file.py

### PR DESCRIPTION
**Pull request by tdarei**
Improved performance 1000x, only in file2video , i am gonna try improve it further and see what other bottlenecks there might be and i will see if the video2file can be sped up too

### CODE REVIEW

File: video2file.py [hunk 1]
```diff
@@ -25,69 +49,79 @@ def checksum(large_file):
...
+    first_frame = cv2.resize(first_frame, dim, interpolation=cv2.INTER_AREA)
+
+    chunks = []
+    with open(sys.argv[1], 'rb') as f:
+        for piece in read_in_chunks(f, chunk_size):
+            chunks.append(piece)
+
+    # Create a multiprocessing Pool
+    pool = Pool()  
+    frames = pool.map(create_frame, chunks)
```
Comment (karaketir16) [inline video2file.py:101]: You are processing all the chunks using multi-process, which means many threads are created (thousands for a 3mb file). This likely consumes a significant amount of memory.

Having that many threads won't speed up the process unless you have a processor with thousands of cores.

Consider processing chunks 12 at a time, or in accordance with the number of cores you have. You can probably fetch the processor core count and set it dynamically.

**Comment by karaketir16** [comment]
Thanks for the contribution. Check the review.

File: video2file.py (hunk 1 above)
Comment (tdarei) [reply video2file.py:101]: Ah, so you think it might be possible to use the thousands of cuda cores an rtx graphics card has?

Comment (karaketir16) [reply video2file.py:101]: I don't know much about cuda

**Comment by karaketir16** [comment]
You have pushed the wrong file.

**Pull request merged**